File: server/python/message/message_creator.py
#!/usr/bin/env python
#-*-coding: utf-8-*-

import os, sys
sys.path.append(os.path.dirname(os.path.realpath(__file__)))
from mysql_holder import MysqlHolder
from mysql_post import MysqlPost
from message import Message

def create_by_mysql(user, password):
    holder = MysqlHolder(user=user, password=password)
    post = MysqlPost(holder.inst())
    return Message(holder, post)

File: server/php/message/message_creator.php
<?php

require_once('mysql_holder.php');
require_once('mysql_post.php');
require_once('message.php');

function createByMysql($user, $password) {
	$holder = new MysqlHolder($user, $password);
    $post = new MysqlPost($holder->inst());
    return new Message($holder, $post);
}

?>
